examples/meta/src/binary/kernel_support_vector_machine.sg

```diff
@@ -11,7 +11,9 @@ Labels labels_test = labels(f_labels_test)
 #![create_features]
 
 #![set_parameters]
+KernelNormalizer n = kernel_normalizer("IdentityKernelNormalizer")
```

Comment:
maybe instead we could have a separate example for under "kernel" for the normalizer?

Reply:
Check plz